test/run-tests

```diff
@@ -216,9 +221,13 @@ def run(*argv, env=None, check=True, cwd=None):
         runenv.update(env)
 
     print("+ " + cwdrepr + envrepr + " ".join(shlex.quote(a) for a in argv))
+    stdout_val = sys.stdout
+    if return_stdout:
+        stdout_val = subprocess.PIPE
 
     return subprocess.run(
-        argv, env=runenv, check=check, stdout=sys.stdout, stderr=sys.stderr, cwd=cwd
+        argv, env=runenv, check=check, stdout=stdout_val, stderr=sys.stderr, cwd=cwd,
+        text=text
```

Comment:
This change was made so that I could use the existing `run` method to get the output of `ansible --version` in order to determine the version of ansible used in the environment.